homeassistant/components/deconz/__init__.py

```diff
@@ -119,6 +119,13 @@ def async_add_remote(sensors):
 
     deconz.start()
 
+    device_registry = await \
+        hass.helpers.device_registry.async_get_registry()
+    device_registry.async_get_or_create(
+        [['bridgeid', deconz.config.bridgeid]], 'Dresden Elektronik',
```

Comment:
This should not be `'bridgeid'` but `DOMAIN`, as it's a unique identifier of the Deconz domain.